src/makeCalculator.js

```diff
@@ -3,8 +3,44 @@
 /**
  * @return {object}
  */
-function makeCalculator() {
-  // write code here
+function makeCalculator(operate) {
```

Comment:
This comment makes sense, please, fix it:

![image](https://github.com/mate-academy/js_advanced_calculator/assets/86923089/5838556e-ef8e-406e-92ff-7a0b019809e8)